src/common/veteran.js

```diff
@@ -259,7 +259,7 @@ const completeVeteran = {
     value: '1235551234',
     dirty: false
   },
-  understandsFinancialDisclosure: true,
+  understandsFinancialDisclosure: makeField(true),
```

Comment:
Ah yes, it should be `true`, you're right on that. But let's not use `makeField` here and instead enter the JSON object directly.